torch/csrc/distributed/c10d/init.cpp

```diff
@@ -617,11 +625,14 @@ pair, :meth:`~torch.distributed.store.get` to retrieve a key-value pair, etc.
 
 Example::
     >>> import torch.distributed as dist
-    >>> server_store = dist.TCPStore("127.0.0.1", 0, True, timedelta(seconds=30))
-    >>> client_store = dist.TCPStore("127.0.0.1", 0, False)
+    >>> from datetime import timedelta
+    >>> # Run on process 1 (server)
+    >>> store = dist.TCPStore("127.0.0.1", 1234, 2, True, timedelta(seconds=30))
+    >>> # Run on process 2 (client)
+    >>> store = dist.TCPStore("127.0.0.1", 1234, 2, False)
     >>> # Use any of the store methods from either the client or server after initialization
-    >>> server_store.set("first_key", "first_value")
-    >>> client_store.get("first_key")
+    >>> store.set("first_key", "first_value")
+    >>> store.get("first_key")
```

Comment:
shall we either keep the naming of the store vars or be clear that `set` is called on server and `get` is called on client?

Reply:
The data is stored on the server, but the `set` and `get` methods can be used both the client and the server. 

That is what I thought the original comment in the docs (`# Use any of the store methods from either the client or server after initialization`) was mentioning, or do I not have the correct understanding? cc: @osalpekar

Reply:
Right, you can call `set` and `get` from both client and sever. It's just a bit clearer if we name them client and server as before to better illustrate that one store was instantiated with `is_server=false`. 

Additionally, this change will actually overwrite the `store` variable to be just the client store, which may make the example more confusing.

Reply:
> Additionally, this change will actually overwrite the store variable to be just the client store, which may make the example more confusing.

The example as it is given doesn't work if all the commands are run together in one file.  Since the world_size is 2 and TCP store will block until all workers have been connected, the client and server need to be instantiated on separate processes. I had named them both `store` since there would only be 1 `store` variable for each process and wanted to show the methods could be called on either process.

But I see what you are saying about the naming and it is confusing, so I will change it back!